commands/actor.go

```diff
@@ -107,6 +110,62 @@ var actorLsCmd = &cmds.Command{
 	},
 }
 
+var actorGetCmd = &cmds.Command{
+	Arguments: []cmdkit.Argument{
+		cmdkit.StringArg("address", true, false, "address to get"),
+	},
+	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
+		addr, err := address.NewFromString(req.Arguments[0])
+		if err != nil {
+			return err
+		}
+		result, err := GetPorcelainAPI(env).ActorGet(req.Context, addr)
+		if err != nil {
+			return err
+		}
+
+		addrStr := addr.String()
+		var output *ActorView
+		switch {
+		case result.Empty(): // empty (balance only) actors have no Code.
+			output = makeActorView(result, addrStr, nil)
```

Comment:
As this code is essentially the same as the code for `actor ls`, let's move it to a private helper for forward-looking maintainability.

Reply:
go-filecoin actor ls --address <actor address>
good idea